app/assets/javascripts/fae/form/inputs/_color.js

```diff
@@ -0,0 +1,41 @@
+/* global Fae, colorpicker, FCH */
+
+/**
+ * Fae form color
+ * @namespace form.color
+ * @memberof form
+ */
+
+Fae.form.color = {
+  init: function() {
+    this.initColorpicker();
+  },
+
+  /**
+   * Initialize colorpicker
+   */
+  initColorpicker: function(e) {
+    var includeAlphaSlider = $('.alpha-slider').length > 0
+
+    var settings = {
+      opacity: includeAlphaSlider,
+      positionCallback: function($elm) {
+        var inputOffset = $elm.offset().top;
+        var distanceToBottom = $(document).height() - inputOffset;
+        var top = distanceToBottom <= this.$UI._height ? 414 : inputOffset + $elm.innerHeight();
+
+        return { left: 30, top: top }
+      },
+      renderCallback: function($elm, toggled) {
+        var colors = this.color.colors;
+
+        // default to HEX if Alpha is 1
+        if ($elm.val() && colors.alpha > 0.9) {
+          $elm.val('#' + colors.HEX);
+        }
+      }
+    }
+
+    $('.color-picker').colorPicker(settings);
```

Comment:
This needs to be a `.js-` prefixed class. The CSS should stylize the unprefixed class, and the JS prefix should manage whatever JS targeting. The element has **both** `js-color-picker color-picker` classes. The extra `color-picker` in `form_helper.rb` changes to `js-color-picker`.

[Previous comment, noting the inverse](https://github.com/wearefine/fae/pull/345/files/274dc0e927e7056cdb718d373047022c5cfe6152#r152445430):
> The purpose of .js- prefixed classes is that they never interact with CSS. This should be a regular class instead (i.e. .input-color-picker or simply .color-picker) and that class name should be included when you're defining input_class on fae_color_picker